pontoon/base/static/js/fluent_interface.js

```diff
@@ -77,6 +77,20 @@ var Pontoon = (function (my) {
     });
   }
 
+  /*
+   * Return a default list of plural forms to be displayed for untranslated strings.
+   * The list if a union of:
+   *   - any custom (numeric) plural forms specified in the original string (bug 1443551)
+   *   - CLDR plurals of the locale
+   */
+  function getPluralForms(element) {
+    var customPluralForms = element.expression.variants.map(function (variant) {
+      return variant.key.value;  // can be undefined, so we filter it out below
+    });
+
+    return customPluralForms.filter(Boolean).concat(Pontoon.locale.cldr_plurals);
```

Comment:
This filters out `0`, too?

Also, would this add `one` to locales that only have `[many]` as plural forms?